js/src/ui/Tooltips/Tooltip/tooltip.js

```diff
@@ -30,15 +31,15 @@ let tooltipId = 0;
 
 class Tooltip extends Component {
   static propTypes = {
-    title: PropTypes.string,
-    text: PropTypes.string,
-    right: PropTypes.bool,
+    className: PropTypes.string,
     currentId: PropTypes.number,
     maxId: PropTypes.number,
-    className: PropTypes.string,
+    right: PropTypes.bool,
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md

Should we ? (PS, `r` is after `o` :laughing:)